hamberger.js

```diff
@@ -0,0 +1,13 @@
+const option = document.querySelector('.option');
+const menu = document.querySelector('.hamberger');
+const closemenu = document.querySelector('.Closebtn');
+
+menu.addEventListener('click', () => {
+  option.style.display = 'flex';
+  menu.style.display = 'block';
+});
+
+closemenu.addEventListener('click', () => {
+  option.style.top = '-100%';
+  closemenu.style.left = '0';
+});
```

Comment:
- [ ] [REQUIRED] Good job so far! Your `hamburger` button is only `open` but I can't `close` it because it doesn't appear, kindly fix it. And your `mobile menu` options are working but when you click one of the `options` mobile menu should disappear but yours isn't, kindly correct it 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/231285053-68702fd4-8fad-434c-b2a8-cb9213635a4a.png)